Monika After Story/game/script-topics.rpy

```diff
@@ -15990,3 +15990,117 @@ label monika_stargazing:
     show monika 5kkbsa
     pause 2.0
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_eyesight",
+            category=["you"],
+            prompt="Wearing glasses",
+            conditional="mas_seenLabels(['monika_player_appearance'])"
+        )
+    )
+
+default persistent._mas_pm_wears_glasses = None
+default persistent._mas_pm_wears_lenses = None
+default persistent._mas_pm_poor_eyesight = None
+
+label monika_player_eyesight:
+    m 3lsb "Hey, [player], I was wondering about something lately."
+    m 1esa "Do you wear glasses, or maybe contact lenses?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Do you wear glasses, or maybe contact lenses?{fast}"
+
+        "I wear glasses.":
+            call monika_player_eyesight_awea(glasses=True)
+
+        "I wear contact lenses.":
+            call monika_player_eyesight_awea(glasses=False)
+
+        "I wear cosmetic glasses.":
+            call monika_player_eyesight_cg
+
+        "I wear glasses occasionally.":
+            call monika_player_eyesight_wgo
+
+        "Nope, I don't.":
+            call monika_player_eyesight_n
+
+    return "derandom"
+
+# Always Wears Eyesight Aids
+label monika_player_eyesight_awea(glasses=False):
+    if glasses:
+        $ persistent._mas_pm_wears_glasses = True
+    else:
+        $ persistent._mas_pm_wears_lenses = True
```

Comment:
Shouldn't we set one of those vars to `False` here if the player doesn't use glasses/lens?

Reply:
Right, I think we should.